src/components/DateRangePicker.js

```diff
@@ -547,7 +582,7 @@ const DateRangePicker = React.createClass({
         fontSize: StyleConstants.FontSizes.MEDIUM,
         paddingLeft: isLargeOrMediumWindowSize ? 10 : 0,
         paddingTop: 10,
-        maxWidth: 250,
+        maxWidth: window.innerWidth > 450 && 250,
```

Comment:
turns out figuring out how to set those default values was harder than I anticipated. I pushed back up with the `&&` I think its fine since the values are just left out when the object is converted, but I'm open to suggestions on alternate solutions!